third-party/py/twitter-commons/src/python/BUCK

```diff
@@ -2,9 +2,6 @@ python_library(
   name = 'twitter-commons',
   base_module = '',
   srcs = glob(['**/*.py']),
-  deps = [
-    '//third-party/py/setuptools:pkg_resources',
-  ],
```

Comment:
I'll add a comment before committing this about this anti-dep, since it appears initially counter-intuitive.